core/app_modules/inventory.js

```diff
@@ -103,6 +102,9 @@ inventory.check_out = co.wrap(function*(items, patron, employee) {
 
       var q = 'INSERT INTO checked_out(call, copy, netid, attendant, due)' +
         'VALUES($1, $2, $3, $4, $5)';
+        
+      var notes ='Item checked out by ' + employee + ' due on ' + due.toString().substring(0,10);
```

Comment:
Does it? That is the first four characters of what it takes I thought. It would be fine to store all of it, I just cut it for the notes section so they don't have to see the long ugly string